### ISSUE TITLE
Potential Hero Idea Thread

### ISSUE THREAD

**Issue by CarpeSwag** [Arhowk/Banjoball#9]
Tiny
Techies (with a spell similar to Zigg's grenade from LoL)
Banjo Smash (hero undetermined)
Banjo Metamorph (Rubick - retheme spell to fit Rubick instead of a shapeshifting reptillian)
Shackleshot (Windranger)
Hookshot (Clockwerk)
Rolling Boulder (Earth Spirit) * we need a new ball model for this
Static Link (Razor)
Time/Speed Bubbles (hero undetermined Faceless Void)
Earth Spike (hero undetermined Lion)
Mobility Wave (hero undetermined, but maybe morphling)
Spooky Scary Skeletons - basically a fiddlesticks fear (hero undetermined)
Personal Forcefield (hero undetermined)
DASH HERO (YEP) (hero undetermined)
Vayne condemn-like ability (hero undetermined)

**Comment by Arhowk** [comment]
No rubik is joker

**Comment by CarpeSwag** [comment]
What was the Joker idea again? I forgot to write it down.

**Comment by Arhowk** [comment]
Because I never told you it

**Comment by CarpeSwag** [comment]
Oh. Right. That would explain that.

**Comment by Arhowk** [comment]
so basically

Sprint - Powershot slide with a random variation of (0..2.5x) the distance. If its less than 0.5, than it just displays like "nope" or something and he doent move at all

Q - Random Ballcarrier Ability - some pool of abilities that are randomly chosen. some could inclode powershot (lol) and like literally put the ball at the chosen point. Unsure if i want to acftually show the player the ability or make it random untill cast

R - Random movement ability - same as previous except movement (e.g. pull, dash)

note- the randoms arent like rndom hero abilities, i want a pool of like 1-2 specific to Joker and maybe like 3-4 of other hero abilties specifically chosen (curve when)

**Comment by Arhowk** [comment]
Sönrîel the Prophet (oracle)

Special Ability - Tear in Causality - spawns a sphere at the target point. Initially, the sphere pushes all units out. Once spawned, the sphere bounces all units off of it like that one game with the ball and the flippers. Additionally, the sphere spawns again at the same spot on the inverted side of the field. If spawned in the goal box / buy the goalie, the radius shrinks drastically. The oracle and all other oracles may move through the sphere freely 

Sprint - Time flux - Consume all mana, teleport on the inverse side of the map. Switches to default sprint when the ball is possessed

**Comment by CarpeSwag** [comment]
Oh whenever you do make your changes, keep in mind I've been writing a lot of shit for like projectiles and the like.

Also, I think having the prophet's ball be slammable and shit would result in some topkek pinball plays. You could probably just spawn them in via the projectile code if you're down for that.

**Comment by Arhowk** [comment]
Lol. I didn't plan on that but it sounds hillarious. The only quirk is that
I planned on using a modified chromosphere particle. I wonder how two of
them back to back will look.

**Comment by CarpeSwag** [comment]
Easy solution. Check for nearby chromospheres. Rebound nearby chromospheres. Suddenly things are flying everywhere.

**Comment by Arhowk** [comment]
No I mean chronospheres are half spheres not full spheres. we could just
give them infinite weight?

**Comment by CarpeSwag** [comment]
Double posting because fkyah:
On a diff note, for DOTA 2 Banjo, as much as it pains me to request this from both you and garf, but can we stick to default DOTA names. I feel like having our own names for heroes would be silly and cause some communication between DOTA 2 players and pure banjoballers (I've met a few). Dota 2 players will refer to the heroes as their DOTA 2 names anyways. Rip the Doctrinal Cake dream

**Comment by Arhowk** [comment]
I disagree. Maintaining default dota names reduces the immersion effect of
the game.  We should be aiming to create something both as functionally and
aesthetically unique as possible from dota (aside from 3D stuff cuz we
poor). Following that logic, why are we changing UIs? default sofa players
don't like a new ui

**Comment by CarpeSwag** [comment]
They're entirely different matters. UI is changed because we don't need half of the DOTA UI and because we'll benefit heavily from a minimalist UI.

I agree with your point on immersion, but we can't escape the fact we're still basically using DOTA 2 heroes for the most part. I know some players actually enjoy the fact the game is like if DOTA 2 heroes were to play football.

If we do do the names, then we gotta go for anagrams m8. It's fookin tradition.

**Comment by Arhowk** [comment]
I'm fine with that, but I still wanna implement role names somehow
(Doctrinal Cake the Supersprint or smth idk)
